packages/core/src/PasteRule.ts

```diff
@@ -167,6 +177,45 @@ export function pasteRulesPlugin(props: { editor: Editor; rules: PasteRule[] }):
   let pasteEvent = typeof ClipboardEvent !== 'undefined' ? new ClipboardEvent('paste') : null
   let dropEvent = typeof DragEvent !== 'undefined' ? new DragEvent('drop') : null
 
+  const processEvent = ({
+    state,
+    from,
+    to,
+    rule,
+    pasteEvt,
+  }: {
+    state: EditorState
+    from: number
+    to: { b: number }
+    rule: PasteRule
+    pasteEvt: ClipboardEvent | null
+  }) => {
+    const tr = state.tr
+    const chainableState = createChainableState({
+      state,
+      transaction: tr,
+    })
+
+    const handler = run({
+      editor,
+      state: chainableState,
+      from: Math.max(from - 1, 0),
+      to: to.b - 1,
+      rule,
+      pasteEvent: pasteEvt,
```

Comment:
Can't we take `pasteEvent` as a variable name and also use a shorthand here?

Reply:
There is already a global variable here with that name: https://github.com/ueberdosis/tiptap/pull/5046/files#diff-6eddae5591c39b131ab5597e8ac574df71fafda96023c4573c45fd73ed066b9fR177